financeager/resources.py

```diff
@@ -120,3 +122,9 @@ class CopyResource(LogResource):
     def post(self):
         args = copy_parser.parse_args()
         return self.run_safely("copy", error_code=404, **args)
+
+
+class VersionResource(Resource):
+    def get(self):
+        val = {'version': __version__}
+        return val
```

Comment:
I think you can shorten this to `return {...}`.
Also please add the missing EOF (it's one thing the travis build complained about).